### ISSUE TITLE
COCO Segmentation Masks Submissions Failing

### ISSUE THREAD

**Issue by animatedbrick** [codalab/codalab-competitions#3408]
Hi,

I have submitted multiple JSON files for the COCO segmentation masks competition which have been successful (100+ submissions) -
https://codalab.lisn.upsaclay.fr/competitions/7383#participate-submit_results

However new submissions (after 19th July) are now all failing giving the error below. 

WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
/opt/conda/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')

I am using the mmdetection framework and training with a model that outputs both object detection and instance segmentation results. The object detection (bounding boxes) submissions are working fine, however the instance segmentation submissions are failing. 

Downloading the output file and looking at the metafile for a failed submission gives the below;

elapsedTime: 268.6641709804535
exitCode: 137
ingestionExitCode: 0

**Comment by animatedbrick** [comment]
Hi Didayolo,

Turns out the issue was the size of the submitted JSON file. They were too large and this caused the failures.

Thanks for your support!

**Comment by xlim1996** [comment]
@animatedbrick Hi, I had the same problem. In my case, the result.json is 1.2GB. I'm unable to compress it to 300MB. Please tell me how you solved the problem.

**Comment by animatedbrick** [comment]
Hi @xlim1996, I'm afraid I did not find a solution for this. 

The best I could do was remove all redundant information from the submission file, and hope it reduced enough to be accepted (I understand this is not ideal). I have used the mmdetection framework. The segmentation submissions include the bounding box coordinates, which I deleted from the json file along with reducing the number of decimal points of the confidence score of the prediction. Going through the old codalab server forums, it appears this was a known issue.
